src/script/lua_api/l_util.cpp

```diff
@@ -349,6 +349,52 @@ int ModApiUtil::l_mkdir(lua_State *L)
 	return 1;
 }
 
+// rmdir(path, recursive)
+int ModApiUtil::l_rmdir(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	const char *path = luaL_checkstring(L, 1);
+	CHECK_SECURE_PATH(L, path, true);
+
+	bool recursive = false;
+
+	if (!lua_isnil(L, 2))
+		recursive = lua_toboolean(L, 2);
```

Comment:
```suggestion
	bool recursive = readParam<bool>(L, 2, false);
```
or similar should work